### DISCUSSION TITLE
Cisco Modular Power Supplies and Fans

### DISCUSSION THREAD

**Discussion by michaelklose** [netbox-community/devicetype-library#1172] [Ideas]
I have seen the last community call on Youtube right now and the short discussion about modular power supplies and fans. Most device types are currently modeled with fixed fans/power supplies as most of them were created before module types were supported. 
In the past I already extracted most uplink modules from Cisco devices to modules.
If there is interest to do this in the same way for fans and power supplies I would open an issue for that and starting to work on it for at least all Cisco devices.

**Comment by danner26** [comment]
We would highly appreciate that! Over the past few months I have been trying to standardize this huge set of data and bring it up to date with current schemas, so any assistance would be majorly appreciated.

**Reply by michaelklose** [reply]
Ok I will open an issue to track that. This might take a while. Please assign it to me. 
If there is anything else where I can help please let me know.

**Reply by danner26** [reply]
Anything else you feel like tackling please feel free to open an issue/pr and ask for it to be assigned to you
